gcloud-java-core/src/main/java/com/google/gcloud/ServiceOptions.java

```diff
@@ -150,6 +155,16 @@ public B serviceRpcFactory(ServiceRpcFactory<ServiceRpcT, OptionsT> serviceRpcFa
       this.serviceRpcFactory = serviceRpcFactory;
       return self();
     }
+
+    public B connectTimeout(int connectTimeout) {
```

Comment:
lets document the units (milliseconds) and suggest "0" for infinite and negative for default settings.